compiler/quilt/nodes.py

```diff
@@ -67,7 +73,18 @@ def __repr__(self):
         data_info = '\n'.join(sorted(self._data_keys()))
         return '%s\n%s%s' % (pinfo, group_info, data_info)
 
-    def _items(self):
+    def _items(self, recursive=False):
```

Comment:
We already have a method that iterates through a package tree in the Package class. Can you use that instead? Also, please keep an eye on PR534.

Reply:
This iterates through a node tree, not a package tree, so it works in cases where a user specifies `"user/package/fizz/bang/buzz"`.

..corresponding with Dima about #534 now.

Reply:
Also -- do you mean in the `quilt.tools.package.Package` class?  I don't see the method for recursively walking the package names/values.

Reply:
We talked on Slack, and you mentioned you were referring to `core.preorder`, but that isn't really applicable to this -- plus, it's a fairly useful public facing feature, as you've noted #534, which is similar (but about values only).

It's probably a good idea to have `Node._keys()`, `Node._values()`, and `Node.items()`, and in this case it's also not reinventing the wheel.

Reply:
This was resolved as "nodes.py is going away".